package.json

```diff
@@ -68,6 +74,8 @@
     "uuid": "^3.3.2"
   },
   "devDependencies": {
+    "@adobe/alloy": "file:./",
```

Comment:
For now this just references the current directory, but when we publish the NPM library I will change this to reference the actual version.